src/back/Project.js

```diff
@@ -9,7 +9,7 @@ var Project = function (config, filepath, options) {
     this.CLASSNAME = 'project';
     ConfigEmitter.call(this, config);
     this.filepath = filepath;
-    this.id = options.id || path.basename(path.dirname(fs.realpathSync(this.filepath)));
+    this.id = String(this.config.parsed_opts.style_name) || options.id || path.basename(path.dirname(fs.realpathSync(this.filepath)));
```

Comment:
Not why we need this call to `String` here ?

Reply:
Because if you pass --style-name 10, it will be a number and script will blow up in some another place where it assumes to be a string.
For example [here](https://github.com/kosmtik/kosmtik/blob/f93b864574da93584d6f0323e46a1426c0fb616e/src/back/Project.js#L25)

Reply:
I'd say this should be handled on the parameter parsing side: have you tried `type: "string"` to force casting ?

Reply:
Yes, you are right.
Moved this to option parser